spec/vcr_configurer.rb

```diff
@@ -47,6 +56,12 @@ def self.filter_header(vcr_config, part, header_name)
         vcr_config.allow_http_connections_when_no_cassette = true
         vcr_config.configure_rspec_metadata!
 
+        vcr_config.debug_logger = File.open('vcr_debug.log', 'w')
+
+        vcr_config.ignore_request do |r|
```

Comment:
warning: Lint/UnusedBlockArgument: Unused block argument - `r`. You can omit the argument if you don't care about it. (rubocop)